app/controllers/journal_entries_controller.rb

```diff
@@ -0,0 +1,31 @@
+class JournalEntriesController < ApplicationController
+    
+  def index
+    @journal_entries = JournalEntry.all
+  end
+  
+  def new
+    @journal_entry = JournalEntry.new
+  end
+  
+  def create
```

Comment:
I can see the Rspec for the controllers is missing. Is there a reason ?

Reply:
yes i haven't integrated rspecs at all, and will continue to do so from now one but there aren't in this PR given its already an older one.